test/test_utils.py

```diff
@@ -291,6 +291,15 @@ def test_get_authenticator_from_credential_file():
     assert authenticator.username == 'my_username'
     del os.environ['IBM_CREDENTIALS_FILE']
 
+    file_path = os.path.join(os.path.dirname(__file__),
+                             '../resources/ibm-credentials-container.env')
+    os.environ['IBM_CREDENTIALS_FILE'] = file_path
+    authenticator = get_authenticator_from_environment('service 1')
+    assert authenticator is not None
+    assert authenticator.token_manager.cr_token_filename == 'crtoken.txt'
+    assert authenticator.token_manager.iam_profile_name == 'iam-user1'
```

Comment:
please add asserts to validate the rest of the properties that you are setting for this service in ibm-credentials-container.env.   I'm especially interested in the disable-ssl property since that is defined as a string in the config property, but the authenticator expects it to be a bool.